modules/rdg/src/userobjects/SlopeLimitingBase.C

```diff
@@ -52,14 +52,6 @@ SlopeLimitingBase::initialize()
   _lslope.clear();
 }
 
-void
-SlopeLimitingBase::threadJoin(const UserObject & y)
-{
-  const SlopeLimitingBase & pps = static_cast<const SlopeLimitingBase &>(y);
-
-  _lslope.insert(pps._lslope.begin(), pps._lslope.end());
```

Comment:
`ElementLoopUserObject` was supposed to run threaded, but I do not think it does. That's why you see the `threadJoin` implemented in all classes that inherit from it.  The threading stuff would need some more work...